driver/src/driver/stablex_driver.rs

```diff
@@ -155,7 +155,8 @@ impl<'a> StableXDriver for StableXDriverImpl<'a> {
             Some(time_limit) => time_limit,
             None => {
                 self.metrics.auction_skipped(batch_to_solve);
-                return DriverResult::Skip(anyhow!("account retrieval exceeded time limit"));
+                warn!("orderbook retrieval exceeded time limit");
```

Comment:
It seems that the "account retrieval exceeded time limit" has become "orderbook retrieval exceeded time limit". Was this intended? Are these the same thing?

Reply:
It was intended. We are calling "`orderbook` retrieval" the process of retrieving the accounts (users + balances) and orders. The orderbook retrieval is what is taking too long and causing this warning message.